app/connectors/httpParsers/EisHttpParsers.scala

```diff
@@ -40,7 +40,7 @@ object EisHttpParsers {
     correlationId: String,
     problem: String,
     status: Int,
-    details: Seq[(JsPath, Seq[JsonValidationError])] = Seq(),
+    details: Seq[(JsPath, Seq[JsonValidationError])],
```

Comment:
Hi ashwini, when run compile it said this default is never used, so basically everywhere use this def has given the parameter rather than blank and use default so I removed the default, run all the tests passed.